build/build.xml

```diff
@@ -310,6 +310,18 @@
 		<delete file="${dir}/vendor/lusitanian/oauth/phpunit.xml.dist" />
 		<delete file="${dir}/vendor/lusitanian/oauth/README.md" />
 
+		<delete dir="${dir}/vendor/ocramius/proxy-manager/tests" />
+		<delete dir="${dir}/vendor/ocramius/proxy-manager/docs" />
+		<delete dir="${dir}/vendor/ocramius/proxy-manager/build" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/.gitignore" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/.travis.yml" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/CONTRIBUTING.md" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpdox.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpmd.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpunit.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/README.md" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/UPGRADE.md" />
```

Comment:
The folder examples and html-docs as well as the files .scrutinizer.yml, .travis.install.sh, index.html, proxy-manager.png, proxy-manager.svg, and STABILITY.md also seem unneeded to me?